Traktor S3: Restructured effect section (#485)
mixxxdj/manual: source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

-------

Because the S3 has limited effects controls, the FX setup is unusual and a little complex.
Each deck has a single effect toggle button (:hwlabel:`FILTER ON`) and one knob, and on the right-hand side of the mixer there are five buttons, one for each effect chain (:hwlabel:`FX SELECT`) and one for the QuickEffect (:hwlabel:`FILTER`).
These buttons and knobs are used in different ways depending on how they are pushed, and together allow the DJ to customize all of the effects.

There are three modes that the effect controls can be in:

#. Filter Mode (initial)
#. Effect Chain Edit Mode

> #. Effect Focus Mode

Author: Sounds reasonable. I added a short description. Since I want to avoid up/down scolling, I kept the description in the section.

> #. Filter Mode (initial) - It is indicated when the :hwlabel:`FILTER ON` buttons have the same colors as the individual decks.
> #. Effect Chain Edit Mode - It is indicated when the :hwlabel:`FILTER ON` buttons are all the same color as one of the effect buttons.
> #. Effect Focus Mode - It is indicated when :hwlabel:`FILTER ON` buttons are all the same color as one of the effects, and one of the :hwlabel:`FX SELECT` buttons is blinking.

Reviewer: ```suggestion
#. Filter Mode (initial): :hwlabel:`FILTER ON` buttons have the same color as the respective decks.
#. Effect Chain Edit Mode: :hwlabel:`FILTER ON` buttons are all the same color as one of the :hwlabel:`FX SELECT` buttons
#. Effect Focus Mode: :hwlabel:`FILTER ON` buttons are all the same color as one of the effects, and one of the :hwlabel:`FX SELECT` buttons is blinking.
```

At any time, you can push the :hwlabel:`FILTER` or :hwlabel:`FX SELECT` button to return to Filter Mode.
If you get lost, try pusing the :hwlabel:`FILTER` button to start over.

> The knobs have Soft Takeover mode enabled, which means you need to turn the physical knob to match the current position of the UI knob before the value will change. If you are wondering why it seems like the values aren't changing, you may need to rotate the knob more.

Reviewer: I think we should add `softTakeover` to https://manual.mixxx.org/2.3/en/glossary.html
I filed https://github.com/mixxxdj/manual/issues/486

Author: For now I do not reference the glossary. I think this should be done in this PR.
